.travis.yml

```diff
@@ -55,30 +51,31 @@ env:
     - SWAGGER_REPO=${MAIN_REPO}#${TRAVIS_BRANCH}:services/swagger
     - SWAGGER_DIR=services/swagger
 
-before_install:
-  - sudo rm /usr/local/bin/docker-compose
-  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
-  - chmod +x docker-compose
-  - sudo mv docker-compose /usr/local/bin
-
-before_script:
-  - export TEST_URL=http://127.0.0.1
-  - export REACT_APP_USERS_SERVICE_URL=http://127.0.0.1
-  - export REACT_APP_EVENTS_SERVICE_URL=http://127.0.0.1
-  - export SECRET_KEY=my_precious
-  - export DISPLAY=:99.0
-  - export DOCKER_ENV=$(if [ "$TRAVIS_BRANCH" == "production" ]; then echo "prod"; else echo "stage"; fi)
-  - sh -e /etc/init.d/xvfb start
-  - sleep 3
-  - docker-compose -f docker-compose-stage.yml up --build -d
-
-script:
-  - bash test.sh $DOCKER_ENV
-
-after_script:
-  - docker-compose -f docker-compose-stage.yml down
-
-after_success:
-  - bash ./docker-push.sh
-  - bash ./docker-deploy-stage.sh
-  - bash ./docker-deploy-prod.sh
+jobs:
+  include:
+    - stage: build docker-compose
+      script:
+      - sudo rm /usr/local/bin/docker-compose
+      - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+      - chmod +x docker-compose
+      - sudo mv docker-compose /usr/local/bin
+    - stage: test
+      script:
+      - export TEST_URL=http://127.0.0.1
+      - export REACT_APP_USERS_SERVICE_URL=http://127.0.0.1
+      - export REACT_APP_EVENTS_SERVICE_URL=http://127.0.0.1
+      - export SECRET_KEY=my_precious
+      - export DISPLAY=:99.0
+      - export DOCKER_ENV=$(if [ "$TRAVIS_BRANCH" == "production" ]; then echo "prod"; else echo "stage"; fi)
+      - sh -e /etc/init.d/xvfb start
+      - sleep 3
+      - docker-compose -f docker-compose-stage.yml up --build -d
+      - bash test.sh $DOCKER_ENV
+    - stage: staging
```

Comment:
Can you rename this step to `cleanup` as it's used to tear down and clean up after the `test` stage